matlab/CARFAC_SAI_hacking.m

```diff
@@ -64,9 +71,5 @@
 %%
 png_name_pattern = 'frames/frame%05d.png';
 MakeMovieFromPngsAndWav(round(frame_rate), png_name_pattern, ...
-  wav_fn, ['CARFAC_SAI_movie_', wav_fn(1:end-4), '.mpg'])
-
-%%
-system('rm -r frames');
-
+  wav_fn, [wav_fn(1:end-4), '.mpg'])
```

Comment:
Can we use something like `fileparts` to bring back the `CARFAC_SAI_movie_` prefix? https://www.mathworks.com/help/matlab/ref/fileparts.html